steam_desktop_updater.py

```diff
@@ -173,7 +173,8 @@ def get_installed_apps(steam_root):
             for app in glob.glob(os.path.join(v, 'steamapps', 'appmanifest_*.acf')):
                 with open(app, 'r') as amf:
                     app_mainfest = vdf.load(amf)
-                    apps.append((v, int(app_mainfest['AppState']['appid'])))
+                    app_state = {k.lower(): v for k, v in app_mainfest['AppState'].items()}
```

Comment:
Please use other (than `k` and `v`) variable names for values in the one-line loop to avoid confusing them with ones from outer loop.
Or, better, rename the `v` in outer loop to something more meaningful, like `folders_path`.